modAionImpl/src/org/aion/zero/impl/StandaloneBlockchain.java

```diff
@@ -364,7 +371,7 @@ public void assertEqualTotalDifficulty() {
 
     @Override
     public synchronized ImportResult tryToConnect(final AionBlock block) {
-        ImportResult result = tryToConnectInternal(block, System.currentTimeMillis() / 1000);
+        ImportResult result = tryToConnectInternalAndSkipTimeCheck(block, System.currentTimeMillis() / 1000);
```

Comment:
I don't think we should make this change. For testing, we can create blocks that would pass the time check using `createNewBlockInternal`.

Reply:
Thanks, didn't know about this option, I will make the change